MediaBrowser.Model/Dlna/ResolutionNormalizer.cs

```diff
@@ -43,7 +43,11 @@ public static ResolutionOptions Normalize(
                 }
             }
 
-            var referenceBitrate = h264EquivalentOutputBitrate * (30.0f / (targetFps ?? 30.0f));
+            var referenceFps = targetFps ?? 30.0f;
```

Comment:
looks good but can you describe the constants here what they are and where they come frome?